README.md

```diff
@@ -64,7 +64,7 @@ You can also use Helm to install Emissary. For more information, see the instruc
 
 Check out full the [Emissary
 documentation](https://www.getambassador.io/docs/emissary/) at
-www.getambassador.io.
+www.getambassador.io/docs/open-source.
 
```

Comment:
I know it's not your change but I think "the" and "full" are inverted here 😄

Reply:
Yeah, I'll fix that now